source/connectors/kafka-sink-data-formats.txt

```diff
@@ -9,10 +9,338 @@ Kafka Sink Connector Data Formats
+       },
+       {
+         "name": "age",
+         "type": "int"
+       },
+       {
+         "name": "active",
+         "type": "boolean"
+       },
+       {
+         "name": "address",
+         "type": {
+           "type": "record",
+           "name": "AddressRecord",
+           "fields": [
+             {
+               "name": "city",
+               "type": "string"
+             },
+             {
+               "name": "country",
+               "type": "string"
+             }
+           ]
+         }
+       },
+       {
+         "name": "food",
+         "type": {
+           "type": "array",
+           "items": "string"
+         }
+       },
+       {
+         "name": "data",
+         "type": {
+           "type": "array",
+           "items": {
+             "type": "record",
+             "name": "Whatever",
+             "fields": [
+               {
+                 "name": "k",
+                 "type": "string"
+               },
+               {
+                 "name": "v",
+                 "type": "int"
+               }
+             ]
+           }
+         }
+       },
+       {
+         "name": "lut",
+         "type": {
+           "type": "map",
+           "values": "double"
+         }
+       },
+       {
+         "name": "raw",
+         "type": "bytes"
+       }
+     ]
+   }
+
+
+.. _avro-logical-types-example:
+
+AVRO Logical Type Example
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following AVRO schema demonstrates use of each supported logical types
+(``Decimal``, ``Date``, ``Time``, and ``Timestamp``):
+
+.. code-block:: json
+
+   {
+     "type": "record",
+     "name": "MyLogicalTypesRecord",
+     "namespace": "com.mongodb.kafka.data.kafka.avro",
+       "fields": [
+         {
+           "name": "myDecimalField",
+           "type": {
+             "type": "bytes",
+             "logicalType": "decimal",
+             "connect.parameters": {
+               "scale": "2"
+             }
+           }
+         },
+         {
+           "name": "myDateField",
+           "type": {
+             "type": "int",
+             "logicalType": "date"
+           }
+         },
+         {
+           "name": "myTimeMillisField",
+           "type": {
+             "type": "int",
+             "logicalType": "time-millis"
+           }
+         },
+         {
+           "name": "myTimeMicrosField",
+           "type": {
+             "type": "long",
+             "logicalType": "time-micros"
+           }
+         },
+         {
+           "name": "myTimestampMillisField",
+           "type": {
+             "type": "long",
+             "logicalType": "timestamp-millis"
+           }
+         },
+         {
+           "name": "myTimestampMicrosField",
+           "type": {
+             "type": "long",
+             "logicalType": "timestamp-micros"
+           }
+         }
+       ]
+     }
+
+.. note::
+
+   If you use AVRO code generation to generate logical types for
```

Comment:
This feels overly wordy. Perhaps

The following type mappings are created when using AVRO code generation for a Java-based producer application. 

Thoughts?

Reply:
In both my existing sentence and your recommended replacement, I spotted what I think is passive voice "Xs are created..."

I will reword this to eliminate passive voice and provide emphasis on the fact that the generated types are not native to Java's system libraries.